From e4f0e495ebdb209debe8331490dca564280cdba2 Mon Sep 17 00:00:00 2001
From: Alison Wang <alison.wang@nxp.com>
Date: Thu, 12 Jul 2018 15:46:14 +0800
Subject: [PATCH 269/706] drm: hdp: Disable the swapping for all the lanes

This patch disables the swapping for all the lanes.

Signed-off-by: Alison Wang <alison.wang@nxp.com>
(cherry picked from commit 49a2d3c2b1120d432e37458964e2601182cf8618)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/gpu/drm/imx/hdp/imx-dp.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/gpu/drm/imx/hdp/imx-dp.c b/drivers/gpu/drm/imx/hdp/imx-dp.c
index 875d619fc6ea..8d652f05d9b8 100644
--- a/drivers/gpu/drm/imx/hdp/imx-dp.c
+++ b/drivers/gpu/drm/imx/hdp/imx-dp.c
@@ -44,10 +44,6 @@ void dp_fw_init(state_struct *state, u32 core_rate)
 
 	ret = CDN_API_General_Test_Echo_Ext_blocking(state, echo_msg, echo_resp,
 		sizeof(echo_msg), CDN_BUS_TYPE_APB);
-
-	/* Line swapping */
-	CDN_API_General_Write_Register_blocking(state,
-		ADDR_SOURCD_PHY + (LANES_CONFIG << 2), 0x0040001b);
 }
 
 static const struct of_device_id scfg_device_ids[] = {
-- 
2.17.1

